components/combobox/stories/combobox.stories.js

```diff
@@ -14,11 +15,11 @@ import { Template, VariantGroup } from "./template.js";
  *
  * ### General notes
  *
- * - Combobox uses `.spectrum-PickerButton` instead of a `.spectrum-Picker`
+ * - Combobox uses `.spectrum-InfieldButton` instead of a `.spectrum-PickerButton`
```

Comment:
This feels like maybe more of a changeset comment? I think the original was meant to be a comment on how the picker differs from combobox. Maybe something like:

```suggestion
 * - Combobox uses `.spectrum-InfieldButton` as a menu trigger
```

or

```suggestion
 * - Combobox uses `.spectrum-InfieldButton` instead of a `.spectrum-Picker`
```

or a combination of those two